src/components/VImageDetails/VRelatedImages.vue

```diff
@@ -6,7 +6,7 @@
     <VLoadingIcon v-if="fetchState.isFetching" />
     <VImageGrid
       :images="media"
-      :show-load-more="false"
+      :is-single-page="true"
```

Comment:
I couldn't find a better name for this prop: it is necessary for the related images, which never have a load more button.
`show-load-more` is very ambiguous, and it can be confused with `canLoadMore`, so I wanted to rename it.